ib_manifest_util/create_hardening_manifest.py

```diff
@@ -47,7 +47,7 @@ def create_local_conda_channel(
     tempfile = Path("temp_env.yaml")
 
     # ensure that conda-forge is the only channel
```

Comment:
Might want to change this comment if we're using more than conda-forge

Reply:
Done.